web/src/components/Table/Row.tsx

```diff
@@ -16,13 +17,20 @@ interface Props {
   ageGroup: number,
 }
 
-const useStyles = makeStyles(() => createStyles({
+interface StyleProps {
+  matches: boolean
+}
+
+const useStyles = makeStyles<Theme, Pick<StyleProps, 'matches'>>((_: Theme) => createStyles({
   red: {
     backgroundColor: COLORS.carnationOverlay,
   },
   green: {
     backgroundColor: COLORS.emeraldOverlay,
   },
+  tableCell: {
+    padding: (props) => (props.matches ? '2px' : 'auto'),
+  },
   name: {
```

Comment:
Any thoughts of hiding few columns with small devices and start showing a more-option button instead?